requirements.txt

```diff
@@ -0,0 +1,9 @@
+cython
+fastparquet
+fire
+flake8
+joblib
+pandas >= 0.25
+pandas-gbq
+pyarrow
+toolz
```

Comment:
I'd recommend pinning all these dependency versions for stability.

Reply:
yeah, definitely. I will do this in a subsequent PR, probably as part of a move to using conda.